### DISCUSSION TITLE
Controlling Table Rows Values

### DISCUSSION THREAD

**Discussion by KrishKash** [iTwin/iTwinUI#1771] [Q&A]
Hi there,

I am trying to use a table with following requirements-
a. Display the data fetched using API call (here 2 rows)
b. Enable adding new row (lets assume 3rd row is added)  at the end which have combobox as the cell type. On selecting an item from 3rd row 1st column- subsequent columns of that row will get options available for selection.

Issue- when i select something  in 3rd row 1st column- it also updates values in 1st and 2nd row which should be fixed irrespective of any changes in 3rd column.

Here is a sandbox for the same- [Controlling Table Rows Values ](https://codesandbox.io/p/sandbox/itwin-ui-react-table-gdvjgg?file=%2Fsrc%2FApp.tsx%3A37%2C1)

I'm not sure if I'm missing something here. Any help is appreciated.

**Comment by mayank99** [comment]
could you provide clear repro steps? i'm not sure i understand the issue.
